main.py

```diff
@@ -34,14 +34,15 @@ def create_menu():
 
 def add_book():
     new_book = ui.get_book_info()
-    try:
+    while new_book:  # if book author and a title exists
```

Comment:
This will always be True. new_book is a Book object and Python considers that to be True